packages/unlock-js/src/Unlock/v11/createLock.js

```diff
@@ -30,9 +30,14 @@ async function _getKeyPrice(lock, provider) {
  * @param {number} _lockVersion the version of the lock to create
  * @param {function} callback invoked with the transaction hash
  */
-export default async function (lock, _lockVersion, callback) {
-  // default lock version to 8
-  const lockVersion = _lockVersion || 8 // TODO: change me to 9 when supported!
+export default async function (lock, lockVersionOrCallback, callback) {
+  // default lock version to 9
+  const lockVersion =
+    typeof lockVersionOrCallback === 'number' ? lockVersionOrCallback : 9
```

Comment:
I think it's better to default to v8 now and then to v9 when we have implemented ask v9 changes

Reply:
oh ok. but the one we just released is v10...

Reply:
ho yeah sorry!